modules/ask_va_api/app/controllers/ask_va_api/v0/address_validation_controller.rb

```diff
@@ -47,7 +47,7 @@ def address_params
       end
 
       def service
-        @service ||= if Flipper.enabled?(:va_v3_contact_information_service, @current_user)
+        @service ||= if Flipper.enabled?(:va_v3_contact_information_service)
                        VAProfile::V3::AddressValidation::Service.new
                      else
                        VAProfile::AddressValidation::Service.new
```

Comment:
AskVAApi::V0::AddressValidationController was updated to mirror V0::Profile::AddressValidationController.

Reply:
Current user was removed from this controller.